packages/siwx/src/configs/CloudAuthSIWX.ts

```diff
@@ -114,22 +130,23 @@ export class CloudAuthSIWX implements SIWXConfig {
     throw new Error(await response.text())
   }
 
-  private getStorageToken(): string | undefined {
-    return localStorage.getItem(this.localStorageKey) || undefined
+  private getStorageToken(key: string): string | undefined {
+    return localStorage.getItem(key) || undefined
   }
 
-  private setStorageToken(token: string): void {
-    localStorage.setItem(this.localStorageKey, token)
+  private setStorageToken(token: string, key: string): void {
+    localStorage.setItem(key, token)
   }
 
-  private clearStorageToken(): void {
-    localStorage.removeItem(this.localStorageKey)
+  private clearStorageTokens(): void {
+    localStorage.removeItem(this.localAuthStorageKey)
```

Comment:
We should use `SafeLocalStorage` utility - it's checking window object for possible SSR issues. Let's switch to it